site/_posts/2022-04-14-announcing-fusionauth-1.36.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: blog-post
+title: Announcing FusionAuth 1.36
+description: This release includes bug fixes, enhancements, and SCIM2 support.
+author: Joyce Park 
+image: blogs/release-1-36/product-update-fusionauth-1-36.png
+category: blog
+tags: topic-troubleshooting scim scim2
+excerpt_separator: "<!--more-->"
+---
+
+We're excited to announce the release of versions 1.36 of FusionAuth. This version shipped April 14, 2022. The 1.36 releases include bug fixes, internal updates, and SCIM support.
+
+<!--more-->
+
+These releases contained features, enhancements, and bug fixes. Please see the [release notes](/docs/v1/tech/release-notes#version-1-36-1) for a full breakdown of the changes between 1.35 and 1.36. 
+
+There are a few improvements that I wanted to call out specifically.
+
+## SCIM 2 support
+
+[SCIM 2](http://www.simplecloud.info) (System for Cross-domain Identity Management), a standardized API for identity management sponsored by the IETF, allows user records to be read, written, and shared via HTTP using a simple JSON schema. In a [long-awaited feature](https://github.com/FusionAuth/fusionauth-issues/issues/106), FusionAuth now supports SCIM formats and operations as well as our own API. Learn more about SCIM support in our [SCIM API documentation](https://fusionauth.io/docs/v1/tech/apis/scim).
```

Comment:
https://fusionauth.io/docs/v1/tech/apis/scim is not live. I'd probably either remove the Learn More or wait until Rob is done and has released the API doc.

Reply:
Got it.